mslib/msui/mpl_qtwidget.py

```diff
@@ -56,12 +56,51 @@
 
 matplotlib.rcParams['savefig.directory'] = LAST_SAVE_DIRECTORY
 
+_DEFAULT_SETTINGS_TOPVIEW = {
+    "draw_graticule": True,
+    "draw_coastlines": True,
+    "fill_waterbodies": True,
+    "fill_continents": True,
+    "draw_flighttrack": True,
+    "draw_marker": True,
+    "label_flighttrack": True,
+    "tov_plot_title_size": "default",
+    "tov_axes_label_size": "default",
+    "colour_water": ((153 / 255.), (255 / 255.), (255 / 255.), (255 / 255.)),
+    "colour_land": ((204 / 255.), (153 / 255.), (102 / 255.), (255 / 255.)),
+    "colour_ft_vertices": (0, 0, 1, 1),
+    "colour_ft_waypoints": (1, 0, 0, 1)}
+
+_DEFAULT_SETTINGS_SIDEVIEW = {
+    "vertical_extent": (1050, 180),
+    "vertical_axis": "pressure",
+    "secondary_axis": "no secondary axis",
+    "plot_title_size": "default",
+    "axes_label_size": "default",
+    "flightlevels": [],
```

Comment:
an old entry was:
https://github.com/Open-MSS/MSS/pull/1705/files#diff-46d3dceb21b8fa62a87ac72721c0c16241048c1eef22ecc21fa3786f282f8f6bL68
"flightlevels": [300, 320, 340]

I'm not verified if that was used, just seen in the review